index.src.html

```diff
@@ -1497,17 +1555,21 @@ <h4 algorithm id="collectfromcredentialstore-federatedcredential">
   </ol>
 
   <h4 algorithm id="create-federatedcredential">
-    `FederatedCredential`'s `[[Create]](options)`
+    `FederatedCredential`'s `[[Create]](origin, options)`
   </h4>
 
-  <dfn for="FederatedCredential" method>\[[Create]](options)</dfn> is called with a
+  <dfn for="FederatedCredential" method>\[[Create]](origin, options)</dfn> is called with 
+  an [=environment settings object/origin=] (|origin|) and a
   {{CredentialCreationOptions}} (|options|), and returns a {{FederatedCredential}} if one can be created,
   `null` otherwise, or an [=exception=] in exceptional circumstances:
 
   <ol class="algorithm">
     1.  Assert: |options|["{{CredentialCreationOptions/federated}}"] [=map/exists=].
 
-    2.  Return the result of executing <a abstract-op>Create a `FederatedCredential` from
+    2.  Set |options|["{{CredentialCreationOptions/federated}}"]'s {{FederatedCredentialInit/origin}}
+        member's value to |origin|'s value.
```

Comment:
Rather than publicly add `origin` to `FederatedCredentialInit`, I'd suggest just passing it to the "Create a `FederatedCredential`" algorithm you adjust below. That is, drop this new step 2, change the existing step 2 to something like the following:

```
2.  Return the result of executing <a abstract-op>Create a `FederatedCredential` from
    `FederatedCredentialInit`</a> on |options|["{{CredentialCreationOptions/federated}}"]
    and |origin|.
```

And note that the relevant algorithm takes an origin.